### ISSUE TITLE
2024-12-10, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#16]
# Today's Arxiv Papers

(1) [Training Large Language Models to Reason in a Continuous Latent Space](http://arxiv.org/abs/2412.06769v1), 2024-12-09

(2) [Why Do Developers Engage with ChatGPT in Issue-Tracker? Investigating Usage and Reliance on ChatGPT-Generated Code](http://arxiv.org/abs/2412.06757v1), 2024-12-09

(3) [Refusal Tokens: A Simple Way to Calibrate Refusals in Large Language Models](http://arxiv.org/abs/2412.06748v1), 2024-12-09

(4) [ONEBench to Test Them All: Sample-Level Benchmarking Over Open-Ended Capabilities](http://arxiv.org/abs/2412.06745v1), 2024-12-09

(5) [JAPAGEN: Efficient Few/Zero-shot Learning via Japanese Training Dataset Generation with LLM](http://arxiv.org/abs/2412.06738v1), 2024-12-09

(6) [AutoDCWorkflow: LLM-based Data Cleaning Workflow Auto-Generation and Benchmark](http://arxiv.org/abs/2412.06724v1), 2024-12-09

(7) [OmniEvalKit: A Modular, Lightweight Toolbox for Evaluating Large Language Model and its Omni-Extensions](http://arxiv.org/abs/2412.06693v1), 2024-12-09

(8) [Exploring Critical Testing Scenarios for Decision-Making Policies: An LLM Approach](http://arxiv.org/abs/2412.06684v1), 2024-12-09

(9) [Toward LLM-Agent-Based Modeling of Transportation Systems: A Conceptual Framework](http://arxiv.org/abs/2412.06681v1), 2024-12-09

(10) [I Don't Know: Explicit Modeling of Uncertainty with an [IDK] Token](http://arxiv.org/abs/2412.06676v1), 2024-12-09

(11) [ILLUME: Illuminating Your LLMs to See, Draw, and Self-Enhance](http://arxiv.org/abs/2412.06673v1), 2024-12-09

(12) [MuMu-LLaMA: Multi-modal Music Understanding and Generation via Large Language Models](http://arxiv.org/abs/2412.06660v1), 2024-12-09

(13) [GEAR: A Simple GENERATE, EMBED, AVERAGE AND RANK Approach for Unsupervised Reverse Dictionary](http://arxiv.org/abs/2412.06654v1), 2024-12-09

(14) [Chatbots im Schulunterricht: Wir testen das Fobizz-Tool zur automatischen Bewertung von Hausaufgaben](http://arxiv.org/abs/2412.06651v1), 2024-12-09

(15) [MAVias: Mitigate any Visual Bias](http://arxiv.org/abs/2412.06632v1), 2024-12-09

(16) [AI TrackMate: Finally, Someone Who Will Give Your Music More Than Just "Sounds Great!"](http://arxiv.org/abs/2412.06617v1), 2024-12-09

(17) [3D Spatial Understanding in MLLMs: Disambiguation and Evaluation](http://arxiv.org/abs/2412.06613v1), 2024-12-09

(18) [Examining the Use and Impact of an AI Code Assistant on Developer Productivity and Experience in the Enterprise](http://arxiv.org/abs/2412.06603v1), 2024-12-09

(19) [Towards Controllable Speech Synthesis in the Era of Large Language Models: A Survey](http://arxiv.org/abs/2412.06602v1), 2024-12-09

(20) [Anchoring Bias in Large Language Models: An Experimental Study](http://arxiv.org/abs/2412.06593v1), 2024-12-09

(21) [Data Quality Enhancement on the Basis of Diversity with Large Language Models for Text Classification: Uncovered, Difficult, and Noisy](http://arxiv.org/abs/2412.06575v1), 2024-12-09

(22) [Applications and Implications of Large Language Models in Qualitative Analysis: A New Frontier for Empirical Software Engineering](http://arxiv.org/abs/2412.06564v1), 2024-12-09

(23) [Challenges and Opportunities for Visual Analytics in Jurisprudence](http://arxiv.org/abs/2412.06543v1), 2024-12-09

(24) [Sloth: scaling laws for LLM skills to predict multi-benchmark performance across families](http://arxiv.org/abs/2412.06540v1), 2024-12-09

(25) [Understanding Factual Recall in Transformers via Associative Memories](http://arxiv.org/abs/2412.06538v1), 2024-12-09

(26) [The Fusion of Large Language Models and Formal Methods for Trustworthy AI Agents: A Roadmap](http://arxiv.org/abs/2412.06512v1), 2024-12-09

(27) [Small Languages, Big Models: A Study of Continual Training on Languages of Norway](http://arxiv.org/abs/2412.06484v1), 2024-12-09

(28) [SafeWorld: Geo-Diverse Safety Alignment](http://arxiv.org/abs/2412.06483v1), 2024-12-09

(29) [Can foundation models actively gather information in interactive environments to test hypotheses?](http://arxiv.org/abs/2412.06438v1), 2024-12-09

(30) [Simulating Human-like Daily Activities with Desire-driven Autonomy](http://arxiv.org/abs/2412.06435v1), 2024-12-09

(31) [Integrating Expert Labels into LLM-based Emission Goal Detection: Example Selection vs Automatic Prompt Design](http://arxiv.org/abs/2412.06432v1), 2024-12-09

(32) [LLM-BIP: Structured Pruning for Large Language Models with Block-Wise Forward Importance Propagation](http://arxiv.org/abs/2412.06419v1), 2024-12-09

(33) [StarWhisper Telescope: Agent-Based Observation Assistant System to Approach AI Astrophysicist](http://arxiv.org/abs/2412.06412v1), 2024-12-09

(34) [GameArena: Evaluating LLM Reasoning through Live Computer Games](http://arxiv.org/abs/2412.06394v1), 2024-12-09

(35) [Exploring Memorization and Copyright Violation in Frontier LLMs: A Study of the New York Times v. OpenAI 2023 Lawsuit](http://arxiv.org/abs/2412.06370v1), 2024-12-09

(36) [World knowledge-enhanced Reasoning Using Instruction-guided Interactor in Autonomous Driving](http://arxiv.org/abs/2412.06324v1), 2024-12-09

(37) [LLaVA-SpaceSGG: Visual Instruct Tuning for Open-vocabulary Scene Graph Generation with Enhanced Spatial Relations](http://arxiv.org/abs/2412.06322v1), 2024-12-09

(38) [PRECISE: Pre-training Sequential Recommenders with Collaborative and Semantic Information](http://arxiv.org/abs/2412.06308v1), 2024-12-09

(39) [DSAI: Unbiased and Interpretable Latent Feature Extraction for Data-Centric AI](http://arxiv.org/abs/2412.06303v1), 2024-12-09

(40) [Beyond pip install: Evaluating LLM Agents for the Automated Installation of Python Projects](http://arxiv.org/abs/2412.06294v1), 2024-12-09

(41) [Mastering Collaborative Multi-modal Data Selection: A Focus on Informativeness, Uniqueness, and Representativeness](http://arxiv.org/abs/2412.06293v1), 2024-12-09

(42) [ZeroKey: Point-Level Reasoning and Zero-Shot 3D Keypoint Detection from Large Language Models](http://arxiv.org/abs/2412.06292v1), 2024-12-09

(43) [S$^{2}$FT: Efficient, Scalable and Generalizable LLM Fine-tuning by Structured Sparsity](http://arxiv.org/abs/2412.06289v1), 2024-12-09

(44) [PediaBench: A Comprehensive Chinese Pediatric Dataset for Benchmarking Large Language Models](http://arxiv.org/abs/2412.06287v1), 2024-12-09

(45) [Methods for Legal Citation Prediction in the Age of LLMs: An Australian Law Case Study](http://arxiv.org/abs/2412.06272v1), 2024-12-09

(46) [iLLaVA: An Image is Worth Fewer Than 1/3 Input Tokens in Large Multimodal Models](http://arxiv.org/abs/2412.06263v1), 2024-12-09

(47) [Optimizing Multi-Task Learning for Enhanced Performance in Large Language Models](http://arxiv.org/abs/2412.06249v1), 2024-12-09

(48) [A Comparative Study of Learning Paradigms in Large Language Models via Intrinsic Dimension](http://arxiv.org/abs/2412.06245v1), 2024-12-09

(49) [Unseen Attack Detection in Software-Defined Networking Using a BERT-Based Large Language Model](http://arxiv.org/abs/2412.06239v1), 2024-12-09

(50) [LLMs as Debate Partners: Utilizing Genetic Algorithms and Adversarial Search for Adaptive Arguments](http://arxiv.org/abs/2412.06229v1), 2024-12-09

(51) [SparseAccelerate: Efficient Long-Context Inference for Mid-Range GPUs](http://arxiv.org/abs/2412.06198v1), 2024-12-09

(52) [Category-Adaptive Cross-Modal Semantic Refinement and Transfer for Open-Vocabulary Multi-Label Recognition](http://arxiv.org/abs/2412.06190v1), 2024-12-09

(53) [Enhancing Adversarial Resistance in LLMs with Recursion](http://arxiv.org/abs/2412.06181v1), 2024-12-09

(54) [AlphaVerus: Bootstrapping Formally Verified Code Generation through Self-Improving Translation and Treefinement](http://arxiv.org/abs/2412.06176v1), 2024-12-09

(55) [Holmes-VAU: Towards Long-term Video Anomaly Understanding at Any Granularity](http://arxiv.org/abs/2412.06171v1), 2024-12-09

(56) [Query-Efficient Planning with Language Models](http://arxiv.org/abs/2412.06162v1), 2024-12-09

(57) [MoSH: Modeling Multi-Objective Tradeoffs with Soft and Hard Bounds](http://arxiv.org/abs/2412.06154v1), 2024-12-09

(58) [Hate Speech According to the Law: An Analysis for Effective Detection](http://arxiv.org/abs/2412.06144v1), 2024-12-09

(59) [MMedPO: Aligning Medical Vision-Language Models with Clinical-Aware Multimodal Preference Optimization](http://arxiv.org/abs/2412.06141v1), 2024-12-09

(60) [AIDE: Task-Specific Fine Tuning with Attribute Guided Multi-Hop Data Expansion](http://arxiv.org/abs/2412.06136v1), 2024-12-09

(61) [Evaluating and Mitigating Social Bias for Large Language Models in Open-ended Settings](http://arxiv.org/abs/2412.06134v1), 2024-12-09

(62) [Privacy-Preserving Large Language Models: Mechanisms, Applications, and Future Directions](http://arxiv.org/abs/2412.06113v1), 2024-12-09

(63) [Infusing Prompts with Syntax and Semantics](http://arxiv.org/abs/2412.06107v1), 2024-12-08

(64) [Enhanced Computationally Efficient Long LoRA Inspired Perceiver Architectures for Auto-Regressive Language Modeling](http://arxiv.org/abs/2412.06106v1), 2024-12-08

(65) [Trust No AI: Prompt Injection Along The CIA Security Triad](http://arxiv.org/abs/2412.06090v1), 2024-12-08

(66) [GraPE: A Generate-Plan-Edit Framework for Compositional T2I Synthesis](http://arxiv.org/abs/2412.06089v1), 2024-12-08

(67) [Mixture-of-PageRanks: Replacing Long-Context with Real-Time, Sparse GraphRAG](http://arxiv.org/abs/2412.06078v1), 2024-12-08

(68) [KaSA: Knowledge-Aware Singular-Value Adaptation of Large Language Models](http://arxiv.org/abs/2412.06071v1), 2024-12-08

(69) [Steering Large Language Models to Evaluate and Amplify Creativity](http://arxiv.org/abs/2412.06060v1), 2024-12-08

(70) [Can Generative AI Solve Your In-Context Learning Problem? A Martingale Perspective](http://arxiv.org/abs/2412.06033v1), 2024-12-08

(71) [Hallucination-aware Optimization for Large Language Model-empowered Communications](http://arxiv.org/abs/2412.06007v1), 2024-12-08

(72) [Does RLHF Scale? Exploring the Impacts From Data, Model, and Method](http://arxiv.org/abs/2412.06000v1), 2024-12-08

(73) [Language hooks: a modular framework for augmenting LLM reasoning that decouples tool usage from the model and its prompt](http://arxiv.org/abs/2412.05967v1), 2024-12-08

(74) [Towards Modeling Human-Agentic Collaborative Workflows: A BPMN Extension](http://arxiv.org/abs/2412.05958v1), 2024-12-08

(75) [Exploring Multi-Grained Concept Annotations for Multimodal Large Language Models](http://arxiv.org/abs/2412.05939v1), 2024-12-08

(76) [Heuristic-Induced Multimodal Risk Distribution Jailbreak Attack for Multimodal Large Language Models](http://arxiv.org/abs/2412.05934v1), 2024-12-08

(77) [Paraphrase-Aligned Machine Translation](http://arxiv.org/abs/2412.05916v1), 2024-12-08

(78) [XKV: Personalized KV Cache Memory Reduction for Long-Context LLM Inference](http://arxiv.org/abs/2412.05896v1), 2024-12-08

(79) [Domain-Specific Translation with Open-Source Large Language Models: Resource-Oriented Analysis](http://arxiv.org/abs/2412.05862v1), 2024-12-08

(80) [Cooperative SQL Generation for Segmented Databases By Using Multi-functional LLM Agents](http://arxiv.org/abs/2412.05850v1), 2024-12-08

(81) [Are Clinical T5 Models Better for Clinical Text?](http://arxiv.org/abs/2412.05845v1), 2024-12-08

(82) [A Self-Learning Multimodal Approach for Fake News Detection](http://arxiv.org/abs/2412.05843v1), 2024-12-08

(83) [A Collaborative Multi-Agent Approach to Retrieval-Augmented Generation Across Diverse Data](http://arxiv.org/abs/2412.05838v1), 2024-12-08

(84) [Large Language Models Merging for Enhancing the Link Stealing Attack on Graph Neural Networks](http://arxiv.org/abs/2412.05830v1), 2024-12-08

(85) [An Entailment Tree Generation Approach for Multimodal Multi-Hop Question Answering with Mixture-of-Experts and Iterative Feedback Mechanism](http://dx.doi.org/10.1145/3664647.3681479), 2024-12-08

(86) [[CLS] Token Tells Everything Needed for Training-free Efficient MLLMs](http://arxiv.org/abs/2412.05819v1), 2024-12-08
